docs/source/glossaries.rst

```diff
@@ -68,6 +71,7 @@ RDMA        Remote Direct Memory Access
 REQ         Request
 rkey        Remote KEY
 RMA         Remote Memory Access
+RNDV        Rendezvous
 RnR         Receiver Not Ready
 RoCE        RDMA over Converged Ethernet
 ROCm        Radeon Open Compute platform(AMD)
```

Comment:
add whitespace between `platform` and `(AMD)`

Reply:
done